README.md

```diff
@@ -1,26 +1,34 @@
 <!-- BEGIN_TF_DOCS -->
 # terraform-module-cloud-gcp-kubernetes-cluster
 
-This Terraform module deploys everything you need in GCP to get a kubernetes cluster up and running. This repo should be used in the context of deploying with an [admiral](https://github.com/glueops/admiral)
+This Terraform module deploys everything you need in GCP to get a kubernetes cluster up and running. This repo should be used in the context of deploying with the [admiral](https://github.com/glueops/admiral) repository.
 
 ## Prerequisites
 
 ### Project Setup
 
-1. Create a **new** GCP Project with the following APIs enabled:
+1. Create a **new** GCP Project with the following APIs enabled: [`cloudresourcemanager.googleapis.com`, `compute.googleapis.com`].  These APIs can be enabled in Cloud Shell using:
 
+```bash
+gcloud services enable cloudresourcemanager.googleapis.com compute.googleapis.com
 ```
-cloudresourcemanager.googleapis.com
-compute.googleapis.com
-```
 
-2. Service Quota increase: `In-use IP addresses (default is 8)`. Have this increased to 64. \_Note: this increase should be immediate, otherwise contact GCP support/account maangement\_
+2. Service Quota increase: `In-use IP addresses (default is 8)` has too low of a default value for deploying Kubernetes and quota should be increased to 64 to create headroom in available IPs.
+Edit the quota for `In-use IP addresses` in the [quota page](https://console.cloud.google.com/iam-admin/quotas) or with the following `gcloud` command.
+
+```bash
+gcloud alpha services quota update \
+    --service=compute.googleapis.com --consumer=projects/antoniostacos-gcp-nonprod \
```

Comment:
should we make `antoniostacos-gcp-nonprod` a more explicit placeholder value?

Reply:
great catch - I missed that one.